app/controllers/orders_controller.rb

```diff
@@ -1,11 +1,32 @@
 class OrdersController < ApplicationController
-  def create
+  def new
     @order = Order.new
+    @order.build_shipping_address
+    @order.build_billing_address
```

Comment:
Nice. I see why this line and the line above it are needed, but is there any particular reason you're not using a decorator for this since you have them?